packages/core/src/index.ts

```diff
@@ -0,0 +1,5 @@
+import Ceramic from "./ceramic";
+
+export * from './store/state-store'
```

Comment:
What do we gain from this export?

Reply:
The ability to import the `StateStore` interface in the `cli` package, where the `S3StateStore` is defined